### PR TITLE
#160751466 Show toaster when notification is sent

### PR DESCRIPTION
**Pull request by d3mola**
#### What does this PR do?
Displays a toast when a user receives a notiication

#### Description of Task to be completed?
- add toast message when notification is sent

#### How should this be manually tested?
- Clone the repo
- Change into the project directory
- Check out this branch with `git checkout ch-show-notification-toast-160751466`
- Start the server `make start`
- Setup the backend by following this [README.md](https://github.com/andela/travel_tool_back)
- In a browser, navigate to `localhost:3000`
- Perform an action that requires you to get a notification(create a request, approve a request)
- You should see a notification

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160751466](https://www.pivotaltracker.com/story/show/160751466)

#### Screenshots (if appropriate)
screen shot 2018-09-26 at 12 26 23 pm

#### Questions:
None

### CODE REVIEW

File: src/helper/socket/socket.js [hunk 1]
```diff
@@ -7,13 +8,13 @@ const baseUrl = resolveBaseUrl();
...
 const serverUrl = baseUrl.replace('/api/v1', '');
 
-const io = socketIOClient(serverUrl, {transports: ['websocket', 'polling']});
+export const io = socketIOClient(serverUrl, {transports: ['websocket', 'polling']});
 
 export default function handleManagerNotification(userId) {
   io.on('notification', (data) => {
     if (data.recipientId === userId) {
       store.dispatch(addNotification(data));
+      toast.success(`${data.senderName} ${data.message}`);
```
Comment (albert85) [inline src/helper/socket/socket.js]: @d3mola Please destructure data.senderName and data.message

Comment (d3mola) [reply src/helper/socket/socket.js]: Got it @albert85

**Comment by d3mola** [comment]
@albert85 there is another story on the PT for the new design.

**Comment by d3mola** [comment]
@Stylll at the moment, notifications only get sent for approvals. While notifications for rejections is a good idea, it's outside the scope of this task.

**Comment by Stylll** [comment]
@d3mola Alright. I didn't know that. Thanks for pointing it out. Great job!.

**Pull request merged**